jekyll/_docs/configuration.md

```diff
@@ -227,13 +214,13 @@ machine:
     version: openjdk7
```

Comment:
I'll get to this once I address that part of the doc. I think GitHub is picking up on some whitespace "changes" right now.